spinegeneric/cli/package_for_correction.py

```diff
@@ -137,7 +137,10 @@ def main():
 
     # Package to zip file
     print("Creating archive...")
-    fname_archive = shutil.make_archive(args.o, 'zip', path_tmp)
+    root_dir_tmp = os.path.split(path_tmp)[0]
+    base_dir_name = args.o.split('/')[-1]
```

Comment:
see my comment --> https://github.com/spine-generic/spine-generic/pull/149#discussion_r461864364

it applies to **everywhere** in the code

Reply:
fixed in https://github.com/spine-generic/spine-generic/pull/149/commits/8f943820971e7e7532d9c43ab5b630e0065f5c35